ScreenManager.c

```diff
@@ -101,7 +104,7 @@ static void checkRecalculation(ScreenManager* this, double* oldTime, int* sortTi
    *timedOut = (newTime - *oldTime > this->settings->delay);
    *rescan = *rescan || *timedOut;
    if (newTime < *oldTime) *rescan = true; // clock was adjusted?
-   if (*rescan) {
+   if (*rescan && !this->state->pauseProcessUpdate) {
       *oldTime = newTime;
       ProcessList_scan(pl);
```

Comment:
No, obviously we need to redraw if we update the processes, but in case we don't we do not need to set `redraw`.
It is set at
https://github.com/htop-dev/htop/blob/55eafd3b39eabdf60b5196d8c728b6eaf190650d/ScreenManager.c#L235-L238